src/styles/_catalog.scss

```diff
@@ -1,22 +1,41 @@
 .catalog {
-  display: flex;
-  flex-wrap: wrap;
+  display: grid;
+  justify-content: center;
+  grid-template-columns: repeat(auto-fill, minmax($card-width, 1fr));
+  gap: 48px;
 
-  max-width: 944px;
-  margin: 0 auto;
+  @media screen and (max-width: 488px) {
+    & {
+      grid-template-columns: repeat(1, $card-width);
+    }
+  }
+
+  @media screen and (min-width: 488px) {
+    & {
+      grid-template-columns: repeat(2, $card-width);
+    }
+  }
+
+  @media screen and (min-width: 768px) {
+    & {
+      grid-template-columns: repeat(3, $card-width);
+    }
+  }
+
+  @media screen and (min-width: 1024px) {
+    & {
+      grid-template-columns: repeat(4, $card-width);
+    }
+  }
```

Comment:
For media use only a min-widht or max-width, but not both. For a mobile-first approach, the min-width from the smallest to largest values. For a PC-first approach, use a max-width from the largest to the smallest

Remove this media

```
@media screen and (max-width: 488px) {
    & {
      grid-template-columns: repeat(1, $card-width);
    }
  }
  ```
  
  and add this rule ` grid-template-columns: repeat(1, $card-width);` to main class `catalog `